src/services/AttendanceService.ts

```diff
@@ -0,0 +1,24 @@
+import { Attendance, AppUser, AppUserRole, Event } from '@Entities';
+import { AttendanceRepositoryToken } from '@Repositories';
+
+import { Repository } from 'typeorm';
+import { singleton, inject } from 'tsyringe';
+
+@singleton()
+export class AttendanceService {
+  private attendanceRepository: Repository<Attendance>;
+
+  constructor(@inject(AttendanceRepositoryToken) attendanceRepository: Repository<Attendance>) {
+    this.attendanceRepository = attendanceRepository;
+  }
+
+  createAttendance(event: Event, attendee: AppUser): Attendance {
```

Comment:
what do you think about renaming this to registerAttendance(event, attendee) and calling attendancerepo.save within the function? I don't see a use case where we createattendances and dont persist them to the db